charts/krill/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
What is the purpose of this pod?

Reply:
Yeah exactly - https://helm.sh/docs/topics/chart_tests/
So you can run `helm test krill` to deploy a test pod, which verifies the svc and deployment have successfully started the pod and it responds. Its part of the default Helm chart set